PyMca5/PyMcaGui/io/QTiledWidget.py

```diff
@@ -464,7 +464,9 @@ def _on_add_clicked(self, *, emit=True):
                     'selection': {'x': channel_sel['x'],
                                   'y': channel_sel['y'],
                                   'm': channel_sel['m'],
-                                  'Channel List': channel_sel['Data Channel List']},
+                                  'Channel List': channel_sel['Data Channel List'],
+                                  'LabelNames': channel_sel['Data Channel List'],
```

Comment:
IIRC, I think you'll need a copy since this list gets mutated in the ScanWindow(?).

```suggestion
                                  'LabelNames': channel_sel['Data Channel List'].copy(),
```

Reply:
Then, again it's probably better to do this in one place, like we did in an earlier iteration within the `TiledDataSource.get_data_object()` method.

https://github.com/padraic-shafer/pymca/blob/6492405ea9b5839271476cb3fb3ab130b368c36d/PyMca5/PyMcaCore/QTiledDataSource.py#L68-L73